command/report/query.go

```diff
@@ -34,7 +34,12 @@ func makeQuery(url string, body []byte, bodyMimeType string) ([]byte, error) {
 	}
 
 	if res.StatusCode >= http.StatusInternalServerError || res.StatusCode != 200 {
-		return resBody, errors.New("Error making the query with HTTP status code: " + strconv.Itoa(res.StatusCode))
+		// Log the response body.
+		if resBody != nil {
+			fmt.Println(string(resBody))
+		}
```

Comment:
@siddhant-deepsource if I'm not wrong, `resBody` would be printed with the error message below already, right? So I guess we need not have a `Println` here?